src/app/app-routing.module.ts

```diff
@@ -1,8 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+import { AdminFormCreateEventComponent } from './components/admin-form-create-event/admin-form-create-event.component';
+import { AdminFormManageEventComponent } from './components/admin-form-manage-event/admin-form-manage-event.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'login', component: LoginComponent },
+  { path: 'register', component: RegisterComponent },
+  { path: 'admin-create', component: AdminFormCreateEventComponent },
```

Comment:
we need to settle on what we are going to call this model, advertisingConfig or something similar, then routes should be 'new/advertising-config'.

Reply:
AdvertisingConfig is good. so the path would be "new/advertisingconfig"?

Reply:
I think the hyphen would be a good idea, so advertising-config